requirements.txt

```diff
@@ -1,6 +1,4 @@
-# These are the packages that need to be installed in order to build HTML
-# documentation of the User's Guide.
-
+natcap.invest
```

Comment:
Since we tend to update the User's Guide `main` branch anytime we make invest changes on `main` (if the changes UG edits) does it make sense to install the latest invest `main` from github instead of the latest release on pypi? 

Stole this from our readthedocs: `pip install "git+https://github.com/natcap/invest@main#egg=natcap.invest"`

Reply:
Sure! In the longer term I wonder if we might start keeping separate "stable" and "latest" versions of the UG.